src/main/java/uk/gov/hmcts/divorce/caseworker/event/CaseworkerReissueApplication.java

```diff
@@ -61,7 +62,8 @@ public void configure(final ConfigBuilder<CaseData, State, UserRole> configBuild
             .grantHistoryOnly(
                 SOLICITOR,
                 SUPER_USER,
-                LEGAL_ADVISOR))
+                LEGAL_ADVISOR,
+                SYSTEMUPDATE))
```

Comment:
system update user is a backend user so do we need to grant history ?

Reply:
It is needed when doing the Case Details filtering of the events, see HasAosDraftedEventPredicate.  We need to know whether there is a draft-aos event and if it is before a caseworker-reissue-application event, where aosIsDrafted field will get reset.